pegasus-shaded-netty/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
```

Comment:
As I was doing this, I realized that the build logic needed to properly shade was already a bit subtle and complicated, so I felt it was better to isolate it to its own subproject, which only has the single purpose of shading Netty properly. Let me know though if you feel strongly about this, though, because with some additional work it should be technically possible to shade this directly into the r2-netty subproject.